docs/swarm/services.md

```diff
@@ -255,6 +255,28 @@ $ docker service create \
 0u6a4s31ybk7yw2wyvtikmu50
 ```
 
+The `--update-failure-fraction` flag controls what fraction of tasks can fail
+during an update before the update as a whole is considered to have failed. For
+example, with `--update-failure-fraction 0.1 --update-failure-action pause`,
+after 10% of the tasks being updated fail, the update will be paused.
+
+An individual task update is considered to have failed if the task doesn't
+start up, or if it stops running within the monitoring period specified with
+the `--update-monitor` flag. The default value for `--update-monitor` is 30
+seconds, which means that a task failing in the first 30 seconds after its
+started counts towards the service update failure threshold, and a failure
+after that is not counted.
+
+## Rolling back to the previous version of a service
+
+In case the updated version of a service doesn't function as expected, it's
+possible to roll back to the previous version of the service using
+`docker service update`'s `--rollback` flag. This will revert the service
+to the configuration that was in place before the most recent
+`docker service update` command. Other options can be combined with
+`--rollback`; for example, `--update-delay 0s` to execute the rollback without
```

Comment:
When other others are combined with rollback are they applied to the service spec, or are they only used for the "update", so only "update" options would work?

My understanding (which may be out of date by now) was that if you ran an update which changed the update config of a service, that change wouldn't be reflected in the current update, it would only apply to the next one, because the service uses the current configuration to perform the update, not the new configuration.

Has that changed? Or is rollback a special case that works differently?